lib/looker.ts

```diff
@@ -40,17 +52,43 @@ export async function runEventCountQuery(filters: any): Promise<any>{
  * @param id the events_count.message_id required for running the looker 
  * query to retrieve CTR metrics
  * @param template the message template
+ * @param channel the normalized channel
+ * @param experiment the experiment slug
+ * @param branch the branch slug
  * @returns a CTR percent value for a message if the Looker query results are
  * defined
  */
-export async function getCTRPercent(id: string, template?: string): Promise<number|undefined> {
-  const queryResult = await runEventCountQuery(
-    { 'event_counts.message_id':  '%' + id + '%' }
-  )
-
-  // XXX fix https://bugzilla.mozilla.org/show_bug.cgi?id=1901036 to remove the 
-  // infobar template condition
-  if (queryResult.length > 0 && template !== 'infobar') {
-    return Number(Number(queryResult[0].primary_rate * 100).toFixed(1))
+export async function getCTRPercent(
+  id: string,
+  template: string,
+  channel?: string,
+  experiment?: string,
+  branch?: string
+): Promise<number | undefined> {
+  let queryResult;
+  if (template === "infobar") {
+    queryResult = await runEventCountQuery(
+      {
+        "messaging_system.metrics__text2__messaging_system_message_id": id,
+        "messaging_system.normalized_channel": channel,
+        "messaging_system.metrics__string__messaging_system_ping_type":
+          template,
+      },
```

Comment:
I would expect the above list of filters to also include an experiment and a branch. Is there some reason there shouldn't be?

Reply:
Yes, this should include those too I missed them here!